packages/metadata-common/src/Tags.tsx

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Dialog, showDialog } from '@jupyterlab/apputils';
+import { checkIcon, addIcon } from '@jupyterlab/ui-components';
+
+import React from 'react';
+
+interface IMetadataEditorTagProps {
+  selectedTags: string[];
+  tags: string[];
+  handleChange: (selectedTags: string[], allTags: string[]) => void;
+}
+
+interface IMetadataEditorTagState {
+  selectedTags: string[];
+  tags: string[];
+  plusIconShouldHide: boolean;
+  addingNewTag: boolean;
+}
+
+/**
+ * CSS STYLING
+ */
+const METADATA_EDITOR_TAG = 'elyra-editor-tag';
+const METADATA_EDITOR_TAG_PLUS_ICON = 'elyra-editor-tag-plusIcon';
+const METADATA_EDITOR_TAG_LIST = 'elyra-editor-tagList';
+
+export class MetadataEditorTags extends React.Component<
```

Comment:
Is there a reason this file is called `Tags` when this class is called `MetadataEditorTags` given this is a single class file the two should match